python/ccxt/test/temp-reserved/old async additionals.txt

```diff
@@ -0,0 +1,163 @@
+async def test_tickers(exchange, symbol):
```

Comment:
is this a leftover?

Reply:
yes, that file https://github.com/ccxt/ccxt/blob/529a1f5ae4675a56eb3c46717a7ee83731ca0892/python/ccxt/test/temp-reserved/old%20async%20additionals.txt is leftover, you can delete it atm.